jaseci_ai_kit/jac_nlp/jac_nlp/cl_summer/cl_summer.py

```diff
@@ -10,8 +10,8 @@
 from sumy.utils import get_stop_words
 import nltk
 
-nltk.download("punkt")
-nltk.download("stopwords")
+nltk.download("punkt", quiet=True)
+nltk.download("stopwords", quiet=True)
```

Comment:
Neccessary?

Reply:
It's not really necessary, but this stop logging while the module download.